package.json

```diff
@@ -50,7 +50,9 @@
 	"scripts": {
 		"test": "grunt"
 	},
-	"dependencies": {},
+	"dependencies": {
+		"jquery": ">= 1.7.0 < 4.0.0"
```

Comment:
I haven't tested locally, but all of the docs for npm and semver show no spaces between the operator and the version, so just to be safe, let's drop the space before `4.0.0`.